src/resssources/FileParagraph.rs

```diff
@@ -0,0 +1,6 @@
+pub struct FileParagraph {
+    files: &str,
```

Comment:
Vec of Pattern?

New struct Pattern { path, is_pattern, is_file}

Reply:
I need some explaination why `is_pattern, is_file`? 
It's boolean values set after analysing the content of the string?

ex: 
- `Files: debian/*`
`Pattern { 'debian/*', true, false}`

- `Files: *`
`Pattern { '*', true, false}`

- `Files: */*.1`
`Pattern { '*/*.1', true, true}`

Reply:
> I need some explaination why is_pattern, is_file?
It's boolean values set after analysing the content of the string?

Yes, that's correct, is_file would be if could find it on the file system. Not sure we should have it for now since it's a bit is_pattern: false